code/game/objects/structures/morgue.dm

```diff
@@ -357,7 +357,7 @@ GLOBAL_LIST_EMPTY(crematoriums)
 /obj/structure/bodycontainer/crematorium/creamatorium/cremate(mob/user)
 	var/list/icecreams = new()
 	for(var/mob/living/i_scream as anything in get_all_contents_type(/mob/living))
-		var/obj/item/food/icecream/IC = new(null, list(ICE_CREAM_MOB = list(null, i_scream.name)))
+		var/obj/item/food/icecream/IC = new /obj/item/food/icecream(null, null, FALSE, /* prefill_flavors = */ list(ICE_CREAM_MOB = list(null, i_scream.name)))
```

Comment:
```suggestion
		var/obj/item/food/icecream/IC = new /obj/item/food/icecream(null, null, FALSE, prefill_flavors = list(ICE_CREAM_MOB = list(null, i_scream.name)))
```

no reason to have it commented out, either do it like this or remove the named arg entirely

Reply:
named args dont work properly in new. they were right the first time to not imply otherwise

that being said i would much rather the `FALSE` be commented than anything

Reply:
@Mothblocks named args work just fine in new

Reply:
you should not be changing the call structure of /New on subtypes